plugin-bazel/src/main/kotlin/org/jetbrains/bazel/utils/BazelWorkingDirectoryManager.kt

```diff
@@ -0,0 +1,12 @@
+package org.jetbrains.bazel.utils
+
+object BazelWorkingDirectoryManager {
```

Comment:
what's the purpose of this class? it looks like a global variable with extra steps...